lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7851,6 +7851,14 @@ static bool isSufficientlyTrivial(const clang::CXXRecordDecl *decl) {
   return true;
 }
 
+static bool hasDefaultArgs(const clang::CXXConstructorDecl *ctor) {
```

Comment:
Nit: `hasDefaultArgs` might not be a good name now. It could be something like `hasNonFirstDefaultArg`.